packages/api/cms-api/src/dam/files/entities/folder.entity.ts

```diff
@@ -65,6 +66,10 @@ export function createFolderEntity({ Scope }: { Scope?: Type<DamScopeInterface>
         @Field()
         archived: boolean;
 
+        @Property({ columnType: "boolean", default: false })
```

Comment:
We typically don't use database default values, as this means we're "lying" in the code. You could instead use a default value in the GraphQL schema

Reply:
Yes, this is better. One downside is the ESLint disable. Maybe we need to configure the rule to not apply for entity fields. And shouldn't the property be in `OptionalProps`?

Reply:
> And shouldn't the property be in OptionalProps?

True

> Maybe we need to configure the rule to not apply for entity fields.

I disabled it for entity files